src/components/Nav.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+const Nav = () => (
+  <div>
+    <ul
+      style={{
+        display: 'flex',
+        listStyle: 'none',
+        gap: '10px',
+      }}
+    >
+      <li>
+        <NavLink activeStyle={{ color: 'orangered' }} to="/home">
+          BOOKS
+        </NavLink>
```

Comment:
- [ ] Please edit this link path because when you click on it there is no route that matches `/home` so it throughs the error below.

![image](https://user-images.githubusercontent.com/68796232/178006210-3f2a4135-8f2f-48cb-94c7-195989d1c0eb.png)

[Hint]: If you make this link path `/` it will match the route you made in App.js and everything will work properly.